app/models/appointment_reminder.rb

```diff
@@ -0,0 +1,16 @@
+class AppointmentReminder < ApplicationRecord
+  belongs_to :appointment
+  belongs_to :patient
+  belongs_to :experiment, class_name: "Experimentation::Experiment", optional: true
```

Comment:
We definitely don't need this. I believe I added it because I expected it would make some sql queries simpler, but I agree that it's redundant information.